store/sqlstore/upgrade.go

```diff
@@ -707,15 +711,29 @@ func UpgradeDatabaseToVersion513(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion514(sqlStore SqlStore) {
-	// TODO: Uncomment following condition when version 5.14.0 is released
-	// if shouldPerformUpgrade(sqlStore, VERSION_5_13_0, VERSION_5_14_0) {
+	if shouldPerformUpgrade(sqlStore, VERSION_5_13_0, VERSION_5_14_0) {
+		saveSchemaVersion(sqlStore, VERSION_5_14_0)
+	}
+}
+
+func UpgradeDatabaseToVersion515(sqlStore SqlStore) {
+	// TODO: Uncomment following condition when version 5.15.0 is released
+	// if shouldPerformUpgrade(sqlStore, VERSION_5_14_0, VERSION_5_15_0) {
+
+	// 	saveSchemaVersion(sqlStore, VERSION_5_15_0)
+	// }
+}
+
+func UpgradeDatabaseToVersion516(sqlStore SqlStore) {
+	// TODO: Uncomment following condition when version 5.16.0 is released
+	// if shouldPerformUpgrade(sqlStore, VERSION_5_15_0, VERSION_5_16_0) {
 
 	if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
 		sqlStore.GetMaster().Exec("ALTER TABLE Tokens ALTER COLUMN Extra TYPE varchar(2048)")
 	} else if sqlStore.DriverName() == model.DATABASE_DRIVER_MYSQL {
 		sqlStore.GetMaster().Exec("ALTER TABLE Tokens MODIFY Extra text")
 	}
```

Comment:
Why have these database changes moved from the 5.14 migration to the 5.16 one?

Reply:
@jespino any comments?

Reply:
Because are related to guest accounts. The change is needed for guest invites because the max length previously was very restrictive. Not needed for 5.14, needed for 5.16.

Reply:
got it.